player.py

```diff
@@ -88,16 +88,38 @@ def choose_ant_to_move(self, allowed_choices):
 
     Parameters
     ----------
-    alowed_choices : (list)
+    allowed_choices : (list)
       A list containing the possible options available to the player.
       Elements are the IDs of the ants as strings.
 
     Returns
     -------
-    user_choice : (string)
+    user_choice_ant : (string)
       The ID of the ant the player has chosen to move
     """
-    self.user_choice = None
-    while self.user_choice not in allowed_choices:
-      self.user_choice = input("Pick something: ")
-    return self.user_choice
\ No newline at end of file
+    self.user_choice_ant = None
+    while self.user_choice_ant not in allowed_choices:
+      self.user_choice_ant = input("Pick something: ")
+    return self.user_choice_ant
+
+  def choose_direction_to_pick_food(self, allowed_choices):
+    """Player chooses which direction relative to the ant's new position
+    to collect food token from
+ 
+   Checks player input against a list of available options.
+ 
+   Parameters
+   ----------
+   allowed_choices : (list)
+     A list containing the possible options available to the player.
+     Elements are the IDs of the ants as strings.
+ 
+   Returns
+   -------
+   user_choice_direction : (string)
+     The user's choice of which direction to collect the food token from.
+   """
+    self.user_choice_direction = None
+    while self.user_choice_direction not in allowed_choices:
+      self.user_choice_direction = input("Pick a direction: ")
+    return self.user_choice_direction
```

Comment:
It turns out this function is almost an exact copy of choose_ant_to_move(). Therefore, there should probably only be one function that gets called with the appropriate choices. For example `choose_an_option()" and then the `input()` also lists the options. This would require unit tests to check that the text displayed in the input shows the options. It's not many lines or code that are duplicate and having two functions with distinct names makes their purpose more clear, so I will leave you to decide whether you want to keep them as two functions or change it to just one. (I realise that you were initially pushing for a single function and I was the one that suggested having several)